packages/next/src/server-build.ts

```diff
@@ -1068,9 +1092,27 @@ export async function serverBuild({
         }
       }
 
+      let launcherData = group.isAppRouter ? appLauncher : launcher;
+      let preloadChunks: string[] = [];
```

Comment:
The preload chunks are version specific, we should either version lock or validate the files exist on disk before adding to the launcher